### DISCUSSION TITLE
Improvement to the Bracket Editor Control Panel

### DISCUSSION THREAD

**Discussion by Alex3753** [ppy/osu#27161] [Ideas]
The Control Panel sidebar, although already a massive improvement from the previous, has a few issues that hold it back from a simple and easy operation.

### Match Time
The Match Time is annoyingly slow to set in this simple Textbox, containing dashes, colons and letters, making every change in schedule an overly cautious procedure.

![image](https://github.com/ppy/osu/assets/66335155/92d3e562-4e01-4de5-aff8-227441a01f26)

As a solution I propose a further simplification of this box, by dividing each part of the match time into its own box, which 
can be filled seperately and be cycled through by pressing TAB or by automatically giving focus to the next box in order.

Very basic example:

https://github.com/ppy/osu/assets/66335155/250692ed-bb08-485b-aed4-f007e80a390e

Another solution would be to ignore the symbols separating the numbers by formatting the inputted value similar to how online shops add spaces between every four digits of one's credit card number, just that our "spaces" are colons, dashes and letters.

### Team Name
The size of the Team Name box becomes a problem once team names get too long and contain vital information in the truncated part of the team name, like in tournaments with multiple teams from the same University

https://github.com/ppy/osu/assets/66335155/2170b788-b905-4a22-b2cf-81ed61e5b09b

As a solution I propose giving the team names in the Dropdown a Tooltip with the full team name, just like they already show a Tooltip with the full team name when you hover over an already selected team.

![image](https://github.com/ppy/osu/assets/66335155/d18e8585-a05a-40b5-9377-80ff27b2b2ab)

I hope these ideas can find their way into lazer sooner or later to help out tournaments big and small 😄

**Comment by peppy** [comment]
For dropdowns, you should be able to search by typing now, so that's one other method that may help you immediately.

**Reply by Joehuu** [reply]
Added truncation/tooltips for dropdown menu items in https://github.com/ppy/osu/pull/27178.

Please post only one thing to track per issue/discussion in the future. I cannot mark anything as an answer as the time UX issue you reported may get lost.

**Reply by Alex3753** [reply]
> For dropdowns, you should be able to search by typing now, so that's one other method that may help you immediately.

Sadly this doesn't work well because of the hotkeys, which have priority over any writing

> Please post only one thing to track per issue/discussion in the future. I cannot mark anything as an answer as the time UX issue you reported may get lost.

Very sorry about this, I thought of these issues at the same time and wrote this, totally slipped my mind these separate ideas should be separated 😞
